quickstart.mdx

```diff
@@ -118,6 +118,8 @@ the basic components of a Modus app and how to deploy it to Hypermode.
     Go to the `index.ts` file and locate the `generateText` function. Modify the function
```

Comment:
This is AssemblyScript specific, so would need to change for the Go example (maybe we use tabs instead of code groups here?)

Reply:
@jacklynlee31 I'm not a fan of transferring the complexity of the platform being polyglot to the user, plus it won't scale with additional languages